packages/e2e-tests/specs/editor/various/__snapshots__/inserting-blocks.test.js.snap

```diff
@@ -88,8 +88,8 @@ exports[`Inserting blocks inserts a block in proper place after having clicked \
 <p>First paragraph</p>
 <!-- /wp:paragraph -->
 
```

Comment:
I think it was a bug in the hook that caused `isDark` attrib to be set to false when a Cover block was first inserted. 

I don't think there are any backwards compat issues to it now behaving correctly and not setting this unless a light image or background is selected after the block is inserted, but let me know if you can see any issues.

I added a Cover block placeholder to a post on trunk and saved, checked out this PR and reloaded the post and there were no errors, and selecting color or image on the placeholder worked as expected.